### ISSUE TITLE
Let keyword breaks legacy browsers

### ISSUE THREAD

**Issue by menelike** [lynckia/licode#909]
Since this code does not use a transpiler and is written in vanilla, `let` should be replaced with `var`

https://github.com/lynckia/licode/blob/master/erizo_controller/erizoClient/src/webrtc-stacks/ChromeStableStack.js#L108

https://github.com/lynckia/licode/blob/master/erizo_controller/erizoClient/src/webrtc-stacks/ChromeStableStack.js#L114

@aalonsog 
I think this was an accident, see https://github.com/lynckia/licode/commit/f4482954f9ff26e9940e765e4dae98120dd2d074

**Comment by jcague** [comment]
We always recommend having the latest browser stable versions to support most of WebRTC. Let was introduced [1] in Chrome 41 (mar 2015) and Firefox 44 (sep 2015). Do you need to use older versions?

[1] https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/let#Browser_compatibility

**Comment by menelike** [comment]
@jcague 

`ErizoClient` is part of our initial page bundle in a bigger project, webRTC communication is one feature we support depending on your browser. 
Without the fix older browsers like Safari on iOS v9 won't be able to load our page (they don't use webRTC) as there is an parsing error. imho this should be ideally handled per transpiling to ES5. 

TLDR: 

It isn't about webRTC support, it's about general browser support when `ErizoClient` is part of a bundle.

**Comment by jcague** [comment]
Actually I think we're currently transpiling it when we generate erizo.js. Anyway, that's not the idea we have in mind and in the future we might well start producing ES6 as the output in erizo.js. We can add an optional parameter when building erizo client to transpile it to ES5.
I'd recommend you to generate a different bundle for legacy browsers, or a external module that optionally adds erizo client based on the browser.

**Comment by menelike** [comment]
@jcague 

I just did not fetch up the latest buildchain and missed https://github.com/lynckia/licode/commit/522b3f658d44dcd9c620a5eaf552ad76f109a587#diff-5ccee31239df1cfcc7e017253f16048b which already solves my concerns.

I'm sorry for this waste of time, we highly appreciate this project, thanks a lot!
